### PR TITLE
added resolved ts related metrics

### PR DESCRIPTION
**Pull request by YuJuncen**
Added the resolved ts related metrics && rearranged some old metrics:
image
image

### CODE REVIEW

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -38518,8 +38518,8 @@
           "targets": [
             {
               "exemplar": true,
-              "expr": "increase(tikv_stream_errors{instance=~\"$instance\"}[45s])",
-              "format": "heatmap",
+              "expr": "irate(tikv_stream_errors{instance=~\"$instance\"}[30s]) * 30",
```
Comment (kennytm) [inline metrics/grafana/tikv_details.json:38521]: yeah this one should still be using `increase` and use `$__rate_interval`

File: metrics/grafana/tikv_details.json [hunk 2]
```diff
@@ -38580,6 +38580,216 @@
...
+              "refId": "A"
+            },
+            {
+              "exemplar": true,
+              "expr": "time() * 1000",
+              "hide": false,
+              "interval": "",
+              "legendFormat": "Current Time",
+              "refId": "B"
+            }
```
Comment (kennytm) [inline metrics/grafana/tikv_details.json]: what are these

File: metrics/grafana/tikv_details.json [hunk 3]
```diff
@@ -38580,6 +38580,216 @@
...
+          },
+          "pluginVersion": "7.5.7",
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "tikv_stream_store_checkpoint_ts{instance=~\"$instance\"}",
+              "format": "time_series",
+              "interval": "",
+              "legendFormat": "{{ task }}",
+              "queryType": "randomWalk",
```
Comment (kennytm) [inline metrics/grafana/tikv_details.json:38768]: randomWalk?

Comment (YuJuncen) [reply metrics/grafana/tikv_details.json:38768]: I'm not sure why, but it did be "randomWalk" when exporting, even the panel looks well... 🙃

image

Comment (YuJuncen) [reply metrics/grafana/tikv_details.json:38768]: (Anyway, I have removed the queryType)

File: metrics/grafana/tikv_details.json (hunk 2 above)
Comment (YuJuncen) [reply metrics/grafana/tikv_details.json]: Those are metrics for the progress of backup. 

(262144 == 2 ^ 18, which transform PD timestamp to physical timestamp.)

**Review by 3pointer — APPROVED** [review]
LGTM

**Review by kennytm — APPROVED** [review]

**Pull request merged**